test/functionalTest/cases/2632_timezones/timezones.test

```diff
@@ -0,0 +1,181 @@
+payload='{
+  "id": "E",
+  "type": "T",
+  "D01": {
+    "value": "2016-10-01T01:23:45.000Z",
+    "type": "DateTime"
+  },
+  "D02": {
+    "value": "2016-10-01T02:23:45.000+01:00",
+    "type": "DateTime"
+  },
+  "D03": {
+    "value": "2016-10-01T02:53:45.000+01:30",
+    "type": "DateTime"
+  },
+  "D04": {
+    "value": "2016-10-01T02:23:45.000+0100",
+    "type": "DateTime"
+  },
+  "D05": {
+    "value": "2016-10-01T02:53:45.000+0130",
+    "type": "DateTime"
+  },
+  "D06": {
+    "value": "2016-10-01T02:23:45.000+01",
+    "type": "DateTime"
+  },
+  "D07": {
+    "value": "2016-10-01T00:23:45.000-01:00",
+    "type": "DateTime"
+  },
+  "D08": {
+    "value": "2016-09-30T23:53:45.000-01:30",
+    "type": "DateTime"
+  },
+  "D09": {
+    "value": "2016-10-01T00:23:45.000-0100",
+    "type": "DateTime"
+  },
+  "D10": {
+    "value": "2016-09-30T23:53:45.000-0130",
+    "type": "DateTime"
+  },
+  "D11": {
+    "value": "2016-10-01T00:23:45.000-01",
+    "type": "DateTime"
+  }
+}'
+orionCurl --url /v2/entities --payload "$payload"
+echo
+echo
+
+
+echo "02. Get entity"
+echo "=============="
+orionCurl --url /v2/entities/E
+echo
+echo
+
+
+--REGEXPECT--
+01. Create entity with several timezones, all meaning the same date
+===================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/E?type=T
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+02. Get entity
+==============
+HTTP/1.1 200 OK
+Content-Length: 835
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "D01": {
+        "metadata": {},
+        "type": "DateTime",
+        "value": "2016-10-01T01:23:45.00Z"
+    },
+    "D02": {
+        "metadata": {},
+        "type": "DateTime",
+        "value": "2016-10-01T01:23:45.00Z"
+    },
+    "D03": {
+        "metadata": {},
+        "type": "DateTime",
+        "value": "2016-10-01T01:23:45.00Z"
+    },
+    "D04": {
+        "metadata": {},
+        "type": "DateTime",
+        "value": "2016-10-01T01:23:45.00Z"
+    },
+    "D05": {
+        "metadata": {},
+        "type": "DateTime",
+        "value": "2016-10-01T01:23:45.00Z"
+    },
+    "D06": {
+        "metadata": {},
+        "type": "DateTime",
+        "value": "2016-10-01T01:23:45.00Z"
+    },
+    "D07": {
+        "metadata": {},
+        "type": "DateTime",
+        "value": "2016-10-01T01:23:45.00Z"
+    },
+    "D08": {
+        "metadata": {},
+        "type": "DateTime",
+        "value": "2016-10-01T01:23:45.00Z"
+    },
+    "D09": {
+        "metadata": {},
+        "type": "DateTime",
+        "value": "2016-10-01T01:23:45.00Z"
+    },
+    "D10": {
+        "metadata": {},
+        "type": "DateTime",
+        "value": "2016-10-01T01:23:45.00Z"
+    },
+    "D11": {
+        "metadata": {},
+        "type": "DateTime",
+        "value": "2016-10-01T01:23:45.00Z"
```

Comment:
From this test case I can see that date string will always be listed referred to UTC, right?